must-gather/collection-scripts/gather_nodes

```diff
@@ -23,6 +23,8 @@ mkdir -p ${NODES_PATH}
 NAMESPACE_MANIFEST="/etc/node-gather/namespace.yaml"
 SERVICEACCOUNT_MANIFEST="/etc/node-gather/serviceaccount.yaml"
 DAEMONSET_MANIFEST="/etc/node-gather/daemonset.yaml"
+CLUSTER_ROLE_PODS_READER="/etc/node-gather/clusterrole.yaml"
```

Comment:
I would have kept (or changed to be consistent) the convention with `_MANIFEST`

Reply:
will change it